services/README.md

```diff
@@ -1,5 +1,15 @@
 # Preferred Services
 
+## Mailing
+[SendGrid](https://sendgrid.com/), in Heroku: use the free Starter addon.
+Remember to set up DKIM (under sender authentication -> domain authentication).
+
+### Mail testing
+Use [mail-tester](https://www.mail-tester.com) to test mail setup.
+
+### Mailtrap
+Use [Mailtrap](https://mailtrap.io) for staging/review-apps.
```

Comment:
I know there are some different opinions about whether we should send out emails or use mailtrap.
So maybe the wording should be less demanding and more optionally.

Reply:
What are the concerns about Mailtrap? (This comment should not be a blocker to merge this PR. I am just curious)

Reply:
Think the difference in opinion is mostly about it being worth it.
The overhead of setting up and using Mailtrap vs just using letter_opener(?) and setting it up properly on production 🤷 🤷‍♂️ 🤷‍♀️

Reply:
In some of our projects, the preferred way to handle mail sending in review apps is to send out mails normally. @heatherabtion
You would then have to ensure that the mail addresses that staging/review apps send to are not problematic.

Reply:
Those are all fair points. I believe the value of Mailtrap is that it makes it easy to use the test environments. E.g. when PM is playing with a test environment, they can see who receives the emails, who is CCed, and so on.

Reply:
@hcarreras I have preferenced using an actual sendgrid setup on master review app within Danish Shipping, and just have an interceptor setup https://guides.rubyonrails.org/action_mailer_basics.html#intercepting-and-observing-emails to redirect to the team's group email. This gives a slightly more realistic indication of mail function within the ecosystem. I would recommend that as it also decreases our external dependencies by one :)

Reply:
One concern I have is that I would like the client to be able to see emails when they test review apps.
I haven't tried Mailtrap, yet...
With team group email the client would be added, which should be fine.

Btw, in case you are not aware, this is a public repository, so don't mention client details.